content/2016/09/30/cssconf.rst

```diff
@@ -0,0 +1,640 @@
+and worth your time
+if you get a chance to watch the videos.
+
+.. _Nicole Sullivan: https://twitter.com/stubbornella
+.. _Bocoup: https://twitter.com/bocoup
+.. _Claudina Sarahe: http://twitter.com/itsmisscs
+.. _slides: https://oddbooksapp.com/book/sass-map-magic
+
+
+Creativity in Programming for Fun and Profit
+--------------------------------------------
+
+.. image:: /static/images/blog/cssconf-2016/sd.jpg
+
+`Sarah Drasner`_
+talked about programming as a creative act.
+There is more than one way to solve any given problem —
+and different solutions will come with different trade-offs.
+It's worth exploring the options,
+while keeping in mind that code is a communication tool,
+and you should engineer with maintainers in mind.
+
+In the second part of the talk,
+Sarah listed several tools for creative thinking.
+I have found all these tools particularly helpful as an artist,
+and it was a nice reminder that they can apply to code as well.
+
+1. Question the base premises
+2. Impose artificial bounds
+3. Repurpose existing solutions
+4. Combine existing solutions
+5. Use open-source code (this seems like an extension of 3 and 4)
+
+Of course,
+this summary barely does justice
+to the beautiful demonstrantions Sarah included.
+Check out her `CodePen`_ page
+for a whole load of amazing and beuatiful samples.
+
+.. _Sarah Drasner: http://twitter.com/sarah_edo
+.. _CodePen: http://codepen.io/sdras/
+
+
+No Bugs in Sight
+----------------
+
+.. image:: /static/images/blog/cssconf-2016/bj.jpg
+
+`Brian Jordan`_
+talked about automated front-end testing,
+with tools like PhantomJS.
+He works with `code.org`_,
+a non-profit working to increase student access
+to computer science curriculum
+across demographics.
+
+I didn't get good notes on his overview of PhantomJS,
+but there were a few philisophical points that stood out:
+
+- Test everything
+- Make sure everyone on the team can write tests
+  for the code they write
+- Take the time, on occasion,
+  to review and clean up your tests —
+  otherwise you will run into bloat and compile-time issues
+
+.. _Brian Jordan: http://twitter.com/bcjordan
+.. _`code.org`: http://code.org
+
+
+Nativize Is the New Normalize
+-----------------------------
+
+.. image:: /static/images/blog/cssconf-2016/jl.jpg
+
+`Jessica Lord`_
+gave one of my favorite talks
+that I didn't understand.
+She covered a lot of ground,
+showing us how to use Electron
+to build native (Mac, Windows, Linux) desktop applications
+using Node, HTML, and CSS —
+with simplified access to native APIs.
+
+That was pretty cool,
+but then she got to the point of the talk title:
+CSS to help "nativize" your styles,
+so your app feels like desktop software,
+and not a website.
+
+Since Electron uses the latest Choromium rendering,
+you don't have to target multiple browsers —
+but you do have to contend with multiple operating systems.
+Among other things,
+she advised disabling
+`cursor: pointer`
+and "rubber-band" scrolling.
```

Comment:
I don't have much. Here's the full block:

```
desktop apps with web technology
- open-source (by GitHub, for Atom)
- windows, mac, linux
- github.com/electron
- not a native wrapper around a web view
- chromium + node + native apis
- one browser, one api
- html, css, js

nativize: remove selection, image dragging, glove-pointer
```

Anything in there you think I should add? Otherwise, I'm just hoping to link all the slides and videos when they are available.